pkg/services/plugindashboards/service/dashboard_updater.go

```diff
@@ -54,7 +54,7 @@ type DashboardUpdater struct {
 func (du *DashboardUpdater) updateAppDashboards() {
 	du.logger.Debug("Looking for app dashboard updates")
 
-	pluginSettings, err := du.pluginSettingsService.GetPluginSettings(context.Background(), 0)
+	pluginSettings, err := du.pluginSettingsService.GetPluginSettings(context.Background(), &pluginsettings.GetArgs{OrgID: 0})
```

Comment:
Nit. Maybe consider `GetAllPluginSettings` or similar without args? Then the consumer doesn't have to provider orgId=0 and logic for "all" can be isolated in plugin settings service.

Reply:
(Discussed offline) Will follow up with this if necessary